skate-plugin/src/main/kotlin/com/slack/sgp/intellij/modeltranslator/GenerateTranslatorBodyAction.kt

```diff
@@ -34,14 +39,35 @@ class GenerateTranslatorBodyAction(private val bundle: TranslatorBundle) : Inten
   override fun startInWriteAction() = true
 
   override fun invoke(project: Project, editor: Editor?, psiFile: PsiFile?) {
+    val startTimestamp = Instant.now()
```

Comment:
It's intentional for the `WhatsNewPanelListener` because I want to measure the session time when the window was initially hidden (the listener is created) and then opened.

For the other parts, I should change them similarly to this PR :). I wasn't particularly interested in tracking the duration for ProjectGen & FeatureFlag Annotation then.